content/user/troubleshooting/blank_page.md

```diff
@@ -28,8 +28,14 @@ installation by using the [Changed Files plugin](https://www.zen-cart.com/downlo
 A PHP syntax error will very often result in a blank screen, or a partially-blank screen.  
 When this happens, there is often also an entry in the server's errorlog, which you can often view via your hosting control panel area. But, sometimes you can't view that log without requesting your hosting company to access it for you, for privacy reasons. So, alternatively you can use the debug logging built-in to Zen Cart, which is described below. 
 
+## 3\. Are you running an older cart on a newer PHP? 
 
-## 3\. Common customization errors
+**Note:** Your hoster may have updated your PHP underneath you; be sure to 
```

Comment:
"underneath you"?

Reply:
While you were looking the other way. :) 

What's your preferred phrase? I thought that was common.